.github/README.md

```diff
@@ -26,7 +26,7 @@ $ make install
 $ cd examples
 ```
 
-4. Inside `main.tf`, set your Management API Key that you can find in your dashboard. If you don't have one, you can generate it.
+4. Inside `main.tf`, set your Management API Key that you can find in [your dashboard](https://app.datadome.co/dashboard/management/integrations). If you don't have one, you can generate it.
```

Comment:
Could you update this "you can generate it", please contact DataDome support to have one generated" (or something like this). It is not opened by default. (Don't ask me why :D)

Reply:
```suggestion
4. Inside `main.tf`, set your Management API Key that you can find in [your dashboard](https://app.datadome.co/dashboard/management/integrations). If you don't have one, please contact DataDome support to generate one.
```

Reply:
Should we specify it to the [public documentation](https://registry.terraform.io/providers/DataDome/datadome/latest/docs#optional) too?